eng/pipelines/cadl-ci.yml

```diff
@@ -70,15 +70,27 @@ stages:
             command: custom
             customCommand: run build
             workingDir: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
+        - pwsh: ./eng/ExecuteCadlEmitterUnitTests.ps1
+          displayName: 'Unit_Test for CADL emitter'
+          workingDirectory: $(Build.SourcesDirectory)
+        - pwsh: |
+            $currentVersion = node -p -e "require('./package.json').version"
+            $devVersion = "$currentVersion-beta.$(Build.BuildNumber)"
+            Write-Host "Setting version to $devVersion"
+            npm version --no-git-tag-version $devVersion | Out-Null;
+          displayName: 'Set Emitter version'
+          workingDirectory: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
+        - script: |
+            dependencyVersion="3.0.0-beta.$(Build.BuildNumber))"
```

Comment:
autorest.csharp minus version may not the same as the emitter minus version

Reply:
And it is better to set the autorest.charp dependency in the package.json with the special version.

Reply:
How could we know the version of autorest.charp